site/en/blog/topics-enhancements-2023-11/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Enhancements to the Topics API
+description: >
+  Updates to top topics selection.
+subhead: >
+  Updates to top topics selection.
+date: 2023-11-02
+thumbnail: 'image/80mq7dk16vVEg8BBhsVe42n6zn82/s3iDQJUgLZV25YbtYxs1.png'
+alt: >
+  Topics API enhancements
+authors:
+  - leeronisrael
+tags:
+  - privacy
+---
+
+
+In June, we outlined several enhancements to the Topics API. We closed by
+reiterating our commitment to continue to listen to ecosystem feedback. Today,
+we are announcing a further enhancement to the Topics API, in response to that
+feedback.
+
+## Top topics selection
+
+The initial Topics API proposal selected users' top five weekly topics based on
+the frequency by which users interacted with each topic, on participating
+websites. We received
+[feedback](https://github.com/patcg-individual-drafts/topics/issues/42) that
+this resulted in the API often returning topics that are less useful for ad
+relevance such as ‘News' or ‘Arts & Entertainment'. We
+[explored](https://github.com/patcg-individual-drafts/topics/issues/42#issuecomment-1029111959)
+many solutions, including allowing callers to set a priority list, ranking by
+inverse frequency on the web, by ad clicks observed by Chrome, and other
+approaches.
+
+The most promising approach we've seen is to integrate Topics utility feedback
+from the advertising ecosystem directly. We have done so by introducing the
+concept of "high utility" buckets. Chrome
+[places](https://docs.google.com/spreadsheets/d/1ZihmbhaTULWdsPkUZlD1vDshbxbMtDwefUsjl96p69o/edit#gid=0)
+each of the 22 root topics (those without an ancestor) from the taxonomy into
+one of two buckets, indicating higher or standard utility for the ecosystem
+overall. All descendants of the root topics inherit the same bucket assignment
+from their parent. The assignment of root topics to buckets  is based on input
+about utility we received from companies across the ecosystem when crafting our
+improved taxonomy.
+
+Considering the above, the updated top topics selection methodology is as
+follows:
+
+1. At the end of each epoch, Chrome converts participating hostnames from
+    the user's browsing history into topics.
+1. First, topics are sorted by bucket, and then by frequency. That is, if
```

Comment:
I might say the following to avoid "first" being the second bullet:
"Then, topics are sorted first by bucket, and then by frequency."

Reply:
Ah — good point. Changed.